services/system/AuthSig/CMakeLists.txt

```diff
@@ -1,9 +1,19 @@
 function(add suffix)
     add_system_service("${suffix}" AuthSig src/AuthSig.cpp)
     add_system_service("${suffix}" RAuthSig src/RAuthSig.cpp)
+    
+    add_library(Spki${suffix} src/Spki.cpp)
```

Comment:
The libraries also need to be installed and there should be a way to refer to them from `psidk`.

Reply:
Okay, I believe this is addressed